qutebrowser/browser/webkit/mhtml.py

```diff
@@ -32,6 +32,7 @@
 import email.encoders
 import email.mime.multipart
 import email.message
+from quopri import encodestring
```

Comment:
Please only import modules (i.e. `import quopri` and use `encodestring.quopri`), as this makes things much more readable IMHO.